### ISSUE TITLE
International language support

### ISSUE THREAD

**Issue by hermanzdosilovic** [saadq/latexresu.me#13]
Hi @saadq, great project! :smile:

I found that some letters, for example `ČĆŽŠĐčćžšđ`, are not displayed properly.

Regards,
Herman

**Comment by mintisan** [comment]
And can't display Chinese for now.

**Comment by saadq** [comment]
Hey @mintisan could you provide a couple of example input characters so that I can test that out as well?

**Comment by psjay** [comment]
+1

 Hope It can support CJK

**Comment by Wesley-Chan** [comment]
@saadq Here are some characters you can use.
Traditional Chinese: 國際語言支持
Simplified Chinese: 国际语言支持

I'm not sure how people display Chinese character with `pdflatex`. I use `xelatex` but it's not compatible with `microtype`, which is used in this project.

**Comment by saadq** [comment]
Some of the templates use `pdflatex`, some use `xelatex`, and some are using `lualatex`. I believe I've got this issue fixed for the `pdflatex` templates by setting the encoding correctly in each template, but I need to play around with the other templates a bit more.

**Comment by bonebizz21** [comment]
Yea

**Comment by MatheusRV** [comment]
and how about i18n?

**Comment by saadq** [comment]
Hey all, I've been working on the resume generator's second version for awhile now. The new version will hopefully have multilanguage support and I will be planning to add i18n to allow the site to work in different languages and such. Looking forward to sharing the new version with you all very soon!

**Comment by saadq** [comment]
Just for a status update – I think I've got the accented characters working for every template. I've also gotten Chinese/Japanese/Korean characters working for the `xelatex` templates with the `xeCJK` package. However, I still need to figure out how to make the C/J/K characters work on `pdflatex` templates. 

Apparently, there is a `CJK` package which is the `pdflatex` version of the `xeCJK`, but I'm not sure how to set my fonts with it With `xeCJK`, all I had to do was:

```latex
\usepackage{xeCJK}
\setCJKmainfont{Source Han Sans}
```

But I get an error when doing:

```latex
\usepackage{CJK}
\setCJKmainfont{Source Han Sans}
```
about `setCJKmainfont` not being a command. Will have to look more into it.


![screen shot 2018-01-02 at 8 24 18 am](https://user-images.githubusercontent.com/5678694/34485128-e5ce8da8-ef96-11e7-9f04-3e9ca409b637.png)

**Comment by Wesley-Chan** [comment]
> make the C/J/K characters work on pdflatex

Thanks for your support. I don't think pdflatex and CJK characters play nicely. That's why basically all Chinese Latex writers are using xelatex.

**Comment by saadq** [comment]
Yes, that makes sense. It's a shame that the majority of the templates I have for the app use `pdflatex`. One thing I can possibly look into is converting them all into `xelatex` but I imagine I will probably need help/pull request for it since I'm not all that confident in my LaTeX skills to be able to do it.

**Comment by Windsooon** [comment]
From what I understand, in template2, template4, template6. we only need to add 

    \usepackage{xeCJK}
    \setCJKmainfont{Source Han Sans}

in their index.js to support Unicode.

**Comment by rudolph9** [comment]
Related is just displaying generic unicode characters. I was trying to add stars next to skills and ran into display issues e.g. `Git ★★★★☆`
